nucleus/security/services/src/main/java/org/glassfish/security/services/common/SecurityAccessFilter.java

```diff
@@ -37,7 +37,7 @@
  * only if the new code is made subject to such option by the copyright
  * holder.
  */
-
+// Portions Copyright [2021] Payara Foundation and/or affiliates
```

Comment:
This class produced many unuseful log lines ...

Reply:
From a developer point of view, I think they are very useful but (most) should not be done at info level.

Reply:
I mean messages like this:
`LOG.fine("The instance has no \'Secure\' annotated ");`

- which instance?